Gems/RecastNavigation/Code/Source/Misc/RecastNavigationPhysXProviderConfig.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Misc/RecastNavigationPhysXProviderConfig.h>
+
+namespace RecastNavigation
+{
+    void RecastNavigationPhysXProviderConfig::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serialize = azrtti_cast<AZ::SerializeContext*>(context))
```

Comment:
nit: `serializeContext` as we updated this some other cases

Reply:
Done